edualgo/backtracking_algorithms/n_queen.py

```diff
@@ -0,0 +1,178 @@
+# N-Queens Problem via Backtracking
+
+from __init__ import print_msg_box
+from time import time as ti
+
+def n_queens(board_size,hint=False):            #Main function argument =size of the board
+
+    st=ti()
+    
+    #Occupied Diagonals and Columns
+    diagonal1={};diagonal2={}                       #For right and left Diagonal respectively
+    Col={}                                          #For Column which are already alloted to some queen 
+    ans=place_queen(0,[],board_size,diagonal1,diagonal2,Col)                      
+    print(ti()-st)
+
+    if not ans:
+        return -1
+
+    if hint:
```

Comment:
This method should be above line 16, isn't it ?

Reply:
yes, got your point.
Anything more to be changed?